libs/tracing/ppx/ppx_tracing.ml

```diff
@@ -54,8 +61,8 @@ let name_of_func_pat (pat : Parsetree.pattern) =
   | Ppat_var { txt; _ } -> txt
   | _ -> "<no func name>"
 
-let tracing_attr (attr : Parsetree.attribute) =
-  if attr.attr_name.txt = "trace" then
+let trace_attr (attr : Parsetree.attribute) =
+  if attr.attr_name.txt = "trace" || attr.attr_name.txt = "trace_debug" then
```

Comment:
If I understand this correctly, I don't really like this approach of setting the level.

Reply:
I agree but let's get this out